sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/io/sermant/flowcontrol/common/xds/retry/RetryConditionManager.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2024-2024 Sermant Authors. All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package io.sermant.flowcontrol.common.xds.retry;
+
+import io.sermant.flowcontrol.common.xds.retry.condition.ConnectErrorRetryCondition;
+import io.sermant.flowcontrol.common.xds.retry.condition.FiveXxErrorCondition;
+import io.sermant.flowcontrol.common.xds.retry.condition.GatewayErrorCondition;
+import io.sermant.flowcontrol.common.xds.retry.condition.ResetBeforeRequestErrorCondition;
+import io.sermant.flowcontrol.common.xds.retry.condition.ResetErrorCondition;
+import io.sermant.flowcontrol.common.xds.retry.condition.SpecificHeaderNameErrorRetryCondition;
+import io.sermant.flowcontrol.common.xds.retry.condition.SpecificStatusCodeErrorRetryCondition;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Retry Evaluator
+ *
+ * @author zhp
+ * @since 2024-11-29
+ */
+public class RetryConditionManager {
+    private static final Map<String, RetryCondition> RETRY_CONDITION_MAP = new HashMap<>();
+
+    static {
+        RETRY_CONDITION_MAP.put("5xx", new FiveXxErrorCondition());
```

Comment:
I think you could use a constant if it's used in many places in your code.

Reply:
done